src/SixLabors.Fonts/Tables/General/Glyphs/GlyphVector.cs

```diff
@@ -23,19 +36,91 @@ internal GlyphVector(Vector2[] controlPoints, bool[] onCurves, ushort[] endPoint
 
         public bool[] OnCurves { get; }
 
-        public Bounds Bounds { get; }
-    }
+        public Bounds Bounds { get; internal set; }
 
-    internal readonly struct GlyphVectorWithColor
-    {
-        internal GlyphVectorWithColor(GlyphVector vector, GlyphColor color)
+        /// <inheritdoc/>
+        public IDeepCloneable DeepClone() => new GlyphVector(this);
+
+        /// <summary>
+        /// TrueType outline, offset glyph points.
+        /// Change the bounding box on the current glyph.
+        /// </summary>
+        /// <param name="dx">The delta x.</param>
+        /// <param name="dy">The delta y.</param>
+        public void TtfOffsetXy(short dx, short dy)
+        {
+            Vector2[] glyphPoints = this.ControlPoints;
+            for (int i = glyphPoints.Length - 1; i >= 0; --i)
+            {
+                glyphPoints[i] = Offset(glyphPoints[i], dx, dy);
+            }
+
+            Bounds orgBounds = this.Bounds;
+            this.Bounds = new Bounds(
+                (short)(orgBounds.Min.X + dx),
+                (short)(orgBounds.Min.Y + dy),
+                (short)(orgBounds.Max.X + dx),
+                (short)(orgBounds.Max.Y + dy));
+        }
+
+        /// <summary>
+        /// Append glyph control points to the current instance.
+        /// </summary>
+        /// <param name="src">The source glyph which points will be appended.</param>
+        public void TtfAppendGlyph(GlyphVector src)
         {
-            this.Vector = vector;
-            this.Color = color;
+            Vector2[] destPoints = this.ControlPoints;
+            int destPointsCount = destPoints.Length;
+            Vector2[] srcPoints = src.ControlPoints;
+            Array.Resize(ref destPoints, destPoints.Length + srcPoints.Length);
+            srcPoints.CopyTo(destPoints.AsSpan(destPointsCount));
         }
 
-        public GlyphVector Vector { get; }
+        public void TtfTransformWith2x2Matrix(float m00, float m01, float m10, float m11)
+        {
+            // Change data on current glyph.
+            // http://stackoverflow.com/questions/13188156/whats-the-different-between-vector2-transform-and-vector2-transformnormal-i
+            // http://www.technologicalutopia.com/sourcecode/xnageometry/vector2.cs.htm
+            float newXmin = 0;
+            float newYmin = 0;
+            float newXmax = 0;
+            float newYmax = 0;
+
+            Vector2[] glyphPoints = this.ControlPoints;
+            for (int i = 0; i < glyphPoints.Length; ++i)
+            {
+                Vector2 p = glyphPoints[i];
+                float x = p.X;
+                float y = p.Y;
+
+                glyphPoints[i] = new Vector2((float)Math.Round((x * m00) + (y * m10)), (float)Math.Round((x * m01) + (y * m11)));
```

Comment:
I'd double check we can't just use the following here. 

https://docs.microsoft.com/en-us/dotnet/api/system.numerics.vector2.transformnormal?view=netframework-4.8

Reply:
please review, if that looks correct: fba5f84d

Reply:
Will do!